src/export_from_inboveg/HT3260_data_preparation.Rmd

```diff
@@ -0,0 +1,297 @@
+  spread(key = var, value = value) %>% 
+  left_join(details_no_survey, by = "recording_givid")
+
+header <- header_3260 %>% 
+  left_join(select(classif_3260, recording_givid, type_observed, classif_key), 
+            by = "recording_givid") %>% 
+  left_join(sitequal, by = c("survey", "recording_givid", "user_reference"))
+
+```
+
+## site
+= collection of structure and layer qualifiers
+
+structure
+```{r}
+structure_3260 %>% select(var_code) %>% distinct()
+
+var_class <- c("profl", "profr", "peil") # peil partim waterpeil
+var_num_elu <- c("dwars", "peil", "artif", "turbi") 
+        # peil partim stroomsnelheid; 
+        # artif: partim lengte van c("Brug", "Bodemplaat of -val", "Duiker")
+        # turbi: partim Secchi
+var_num <- c("schab", "schao") 
+var_multiple <- c("beh", "orgma", "artif", "turbi", "wijze")
+
+structure_class <- structure_3260 %>% 
+  filter(var_code %in% var_class,
+         !(var_value_code %in% c("SDiep", "SStr"))) %>% # exclude data stroomsnelheid (=numeric)
+  select(survey, recording_givid, user_reference, var, value_class = var_description_or_value)
+  
+structure_num_elu <- structure_3260 %>% 
+  filter(var_code %in% var_num_elu) %>% 
+  mutate(value_num = as.numeric(str_replace(value, ",", "\\.")),
+         var = ifelse(is.na(var_description_or_value), var_value_code, var_description_or_value)) %>% 
+  filter(!is.na(value_num)) %>% 
+  select(survey, recording_givid, user_reference, var, value_num)
+
+# tmp - to do: opvolgen of NA's in description aangevuld zijn
+structure_3260 %>% 
+  filter(is.na(var_description_or_value),
+         !(var_value_code %in% c("SDiep", "SStr"))) %>% 
+  select(survey, recording_givid, user_reference, var_value_code, var_description_or_value) # 1 record
+
+structure_num <- structure_3260 %>% 
+  filter(var_code %in% var_num) %>% 
+  mutate(value_num = as.numeric(var_description_or_value)) %>% 
+  select(survey, recording_givid, user_reference, var, value_num)
+  
+structure_multiple <- structure_3260 %>% 
+  filter(var_code %in% var_multiple,
+         !(var_description_or_value %in% c("Secchi-diepte in m", "Secchi-diepte meer dan (m)"))) %>%
+  select(survey, recording_givid, user_reference, var, value_class = var_description_or_value)
+# to do: wijze hoort mss bij header; maar is multiple value (max 3)
+
+structure <- bind_rows(structure_class, structure_multiple, structure_num_elu, structure_num)
+
+```
+
+layerqual
+```{r}
+layerqual_3260 %>% 
+  select(value, value_perc) %>% 
+  arrange(value) %>% distinct()
+# to do: 0,1 in inboveg laten aanpassen, want levert NA op
+
+layerqual <- layerqual_3260 %>% 
+  mutate(value_num = as.numeric(value_perc)) %>% 
+  select(survey, recording_givid, user_reference, var, value_num)
+# values < 1% are converted to 0.5 %
+
+```
+
+combined: site = structure + layer
+```{r}
+site <- bind_rows(structure, layerqual)
+  
+```
+
+@Toon: opbouw data.frame 'site' ok? of value_num & value_class te combineren?
```

Comment:
structuur lijkt me OK. Wel enkele opmerkingen.

- Nu heb je voor bv. survey WarBe_04_20200803 deze twee records:
     - var = artificiële structuren; var_class = Brug
     - var = Brug; var_value = 1.5
     - Ik zou dat in één record steken:  var = artificiële structuren; var_class = Brug; var_value = 1.5
- Submerse vegetatie: misschien beter var = 'bedekking submerse vegetatie' en de verschillende klassen onder value_class 
- Deze variabelen zijn niet echt duidelijk voor mij: "Secchi-diepte in m" , "Slib meer dan (m)" , "Diepte meer dan (m)",  "Secchi-diepte meer dan (m)", "SDiep", "SStr"

Reply:
- artificiële structuren num_class en num_value in 1 record: ok; aangepast
- submerse vegetatie: ok; aangepast
- "SDiep" en "SStr" aangevuld in code; maar zal later in inboveg aangepast worden
- Secchi-diepte in m" , "Slib meer dan (m)" , "Diepte meer dan (m)", "Secchi-diepte meer dan (m)" zal later aangevuld worden